.github/workflows/staging-push.yml

```diff
@@ -42,14 +42,16 @@ jobs:
             echo "TESTRAIL_HOST=${{ secrets.TESTRAIL_HOST_TEST }}" >> $GITHUB_ENV
             echo "TESTRAIL_USERNAME=${{ secrets.TESTRAIL_USERNAME }}" >> $GITHUB_ENV
             echo "TESTRAIL_PASSWORD=${{ secrets.TESTRAIL_PASSWORD }}" >> $GITHUB_ENV
+            echo "JIRA_HOST=${{ secrets.JIRA_HOST }}" >> $GITHUB_ENV
+            echo "JIRA_USER=${{ secrets.JIRA_USER }}" >> $GITHUB_ENV
+            echo "JIRA_PASSWORD=${{ secrets.JIRA_PASSWORD }}" >> $GITHUB_ENV
             echo "GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_ENV
 
       #- name: Update DB - test runs 
       #  run: python ./__main__.py --report-type test-run-counts --project fenix --start-date 2021-08-15 --end-date 2021-10-01 
 
-      - name: Update DB - test coverage
-        #run: python ./__main__.py --report-type test-case-coverage --project ALL
-        run: python ./__main__.py --report-type test-case-coverage --project fenix
+      - name: Jira query
+        run: python ./__main__.py --report-type jira
```

Comment:
this is good for now.  however, we may eventually have more than one report type for jira so this might be better as something like "jira-qa-requests"